.github/workflows/ci.yaml

```diff
@@ -22,20 +22,18 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
+          - '3.12'
           - '3.11'
           - '3.10'
           - '3.9'
           - '3.8'
         airflow-version:
-          - '2.7.2'
-          - '2.6.3'
-          - '2.5.3'
-          - '2.4.3'
+          - '2.9.2'
+          - '2.8.4'
+          - '2.7.3'
         dbt-version:
+          - 1.8
           - 1.7
-          - 1.6
-          - 1.5
-          - 1.4
         exclude:
           # Incompatible combinations
           - python-version: 3.11
```

Comment:
If we go ahead with the changes above, should be fine to remove these two excludes.